src/main/java/com/skankhunt220/config/UserAutoConfiguration.java

```diff
@@ -0,0 +1,10 @@
+package com.skankhunt220.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+@Configuration
+@EnableMongoRepositories(basePackages = "com.skankhunt220.repository.user", mongoTemplateRef = "primaryMongoTemplate")
```

Comment:
Move on this config configurations of primaryMongoTemplate, it will be more readible